### ISSUE TITLE
Documentation: Using Mesh Template

### ISSUE THREAD

**Issue by strarsis** [linchpinagency/mesh#58]
I created a Mesh Template and now I want to apply it to a page.
Apparently there is no documentation on this yet. 
There seems to be no button one can use in order to select a template.
Will the sections on a page that have been applied by a template change when the template changes?

**Comment by aaronware** [comment]
Good point about the documentation.  At this point Templates are JUST starting points of a page. So if you go to a fresh page (one that does not have any mesh sections). When you go to add a new section it will give you the option to start from a template.

In the near future you will have the ability to apply template changes to pages that are using the selected template.
